<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicalDisordersFormClass</class>
 <widget class="QMainWindow" name="MedicalDisordersFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedicalDisordersForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetDisorders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetSymptoms">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search by category:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search by name::</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearchCategory">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearchName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>291</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>See symptoms </string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert the disorder name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDisorderName">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowSymptoms">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Show symptoms</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNotFound">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MedicalDisordersForm.qrc"/>
 </resources>
 <connections/>
</ui>
